### PR TITLE
Start building solution without VS

### PR DESCRIPTION
**Pull request by rstm-sf**
## What does the pull request do?
Start building solution without VS

## What is the current behavior?
In order to build a solution using Nuke, we need a pre-installed msbuild from VS

## What is the updated/expected behavior with this PR?
Use msbuild from dotnet sdk

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Avaloniaui.net with user documentation

### CODE REVIEW

File: nukebuild/Build.cs [hunk 1]
```diff
@@ -39,21 +39,9 @@ partial class Build : NukeBuild
...
-            string msBuildExe = Path.Combine(msBuildDirectory, @"MSBuild\Current\Bin\MSBuild.exe");
-            if (!System.IO.File.Exists(msBuildExe))
-                msBuildExe = Path.Combine(msBuildDirectory, @"MSBuild\15.0\Bin\MSBuild.exe");
-
-            return msBuildExe;
-        }
-
-        return null;
+        return RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+            ? RootDirectory / ".tmp" / "dotnet-win" / "sdk" / "3.1.401" / "MSBuild.dll"
```
Comment (rstm-sf) [inline nukebuild/Build.cs:43]: It would be good to think about not hard-coding the version

**Comment by kekekeks** [comment]
MSBuild from .NET Core can't build Xamarin projects

File: nukebuild/Build.cs (hunk 1 above)
Comment (maxkatz6) [reply nukebuild/Build.cs:43]: I think you should be able to read global.json file here to find dotnet sdk version.

**Comment by rstm-sf** [comment]
> MSBuild from .NET Core can't build Xamarin projects

😒 https://stackoverflow.com/questions/62211169/building-xamarin-android-app-on-windows-no-visual-studio-build-agent-setup#comment110034053_62211169

**Comment by rstm-sf** [comment]
Then it seems to me that it should be postponed

**Pull request closed without merging**